Makefile.include

```diff
@@ -93,14 +97,22 @@ ifneq ($(abspath $(RIOTBASE)/boards),$(abspath $(RIOTBOARD)))
   __DIRECTORY_VARIABLES += RIOTBOARD
 endif
 
-ifneq (,$(BOARDSDIR))
-  # Only warn users, not the CI.
-  ifneq ($(RIOT_CI_BUILD),1)
-    # Do not warn when set from sub-make
-    ifeq ($(MAKELEVEL),0)
+# Only warn users, not the CI.
+ifneq ($(RIOT_CI_BUILD),1)
+  # Do not warn when set from sub-make
+  ifeq ($(MAKELEVEL),0)
+    ifneq (,$(BOARDSDIR))
       $(warning Using BOARDSDIR is deprecated use EXTERNAL_BOARD_DIRS instead)
       $(info EXTERNAL_BOARD_DIRS can contain multiple folders separated by space)
     endif
+
+    # API change warning for EXTERNAL_MODULE_DIRS, remove by 2021.10
+    ifneq (,$(EXTERNAL_MODULE_DIRS))
+      ifeq (1,$(WARNING_EXTERNAL_MODULE_DIRS))
```

Comment:
Does this make sense now? I had added it so that a user that had migrated could disable the warning, but maybe it does not make sense.

Reply:
I think power users will be happy to disable the warning, once the have fully migrated :-)

Reply:
Ahh, I was just confused because I didn't see the definition, but it's just above, isn't it better to keep it here with where its used (feel free to dismiss, just commenting since with a red Murdock this will require another build anyway)